README.md

```diff
@@ -111,8 +111,8 @@ See the [contribution guide][contributing-guidelines] for the contribution proce
 [the development guidelines][development-guidelines] for further
 details on coding in Celeritas, and [the administration guidelines][administration-guidelines] for community standards and roles.
 
-[contributing-guidelines]: https://celeritas-project.github.io/celeritas/user/appendix/development.html#contributing-to-celeritas
-[development-guidelines]: https://celeritas-project.github.io/celeritas/user/appendix/development.html#code-development-guidelines
+[contributing-guidelines]: https://celeritas-project.github.io/celeritas/user/development/contributing.html
+[development-guidelines]: https://celeritas-project.github.io/celeritas/user/development/coding.html
```

Comment:
Thanks for the corrections @pcanal . Buuuut in accordance with the guidelines…
> All changes to the codebase must go through the pull request, but due to the overhead of reviewing, testing, merging, and documenting a PR, we'd like to avoid small changes that have almost no effect in terms of operation or readability. For example, if you find a typo in the documentation, check the rest of the docs for any other typos or improvements you'd like to make, and submit a single PR with those changes.

…I was taking another look at the readme and saw at least one other inconsistency: "Follow the first two steps above to install Spack" whereas there's no steps above. Since you're modifying this file could you please take a look at the rest of the readme and verify there's nothing wrong/broken/incomplete?

Reply:
Also further down is a paragraph on citation (now somewhat a duplicate of Citation.cff), do we still need it here?

Reply:
I think it's important to have in the front-facing readme, but yes I should've updated it to mention the CITATION.cff file. Or maybe we should avoid the duplication and just provide a direct link `[citation](./CITATION.cff)` and a mention of the button on the repository sidebar?